fibonacci-sequence/README.md

```diff
@@ -0,0 +1,30 @@
+Fibonacci Sequence
```

Comment:
move inside fibonacci directory, rename the file if you want, but keep the directory fibonacci.